README.md

```diff
@@ -13,37 +13,41 @@ even Lisp. High-order functions such as ``map``, ``filter``, ``reduce``,
 
 Let's see an example:
 
-    > -- Functional style
-    > require "fun" ()
-    > -- calculate sum(x for x^2 in 1..n)
-    > n = 100
-    > print(reduce(operator.add, 0, map(function(x) return x^2 end, range(n))))
-    328350
-
-    > -- Object-oriented style
-    > local fun = require "fun"
-    > -- calculate sum(x for x^2 in 1..n)
-    > print(fun.range(n):map(function(x) return x^2 end):reduce(operator.add, 0))
-    328350
+```lua
+> -- Functional style
+> require "fun" ()
```

Comment:
AFAIK it is an "old" style for `require` in Lua. At least in Tarantool we don't use such style. Could you replace it with `require('fun')`?

https://www.tarantool.io/en/doc/latest/dev_guide/lua_style_guide/#idioms-and-patterns

Reply:
I'll look around (but today's time is spent now).

Reply:
This project consistently uses `require 'fun'` and `require 'fun' ()`. I cannot advocate it by my preference and another project guidelines. I'll leave it as is for now. Feel free to propose such change if there is a real reason aside preference of majority of authors.

Reply:
Okay, let's keep it as is.